tests/integration-tests/tests/create/test_create.py

```diff
@@ -23,13 +23,18 @@
 
 @pytest.mark.dimensions("eu-central-1", "c5.xlarge", "ubuntu1804", "*")
 @pytest.mark.usefixtures("instance", "scheduler")
-def test_create_wrong_os(region, os, pcluster_config_reader, clusters_factory, architecture):
+def test_create_wrong_os(region, os, pcluster_config_reader, clusters_factory, architecture, amis_dict):
     """Test error message when os provide is different from the os of custom AMI"""
     # ubuntu1804 is specified in the config file but an AMI of centos7 is provided
     wrong_os = "centos7"
     logging.info("Asserting os fixture is different from wrong_os variable")
     assert_that(os != wrong_os).is_true()
-    custom_ami = retrieve_latest_ami(region, wrong_os, ami_type="pcluster", architecture=architecture)
+    if amis_dict:
+        custom_ami = amis_dict.get(wrong_os)
+    else:
+        custom_ami = retrieve_latest_ami(
```

Comment:
shall we replace any other usage of retrieve_latest_ami with the new fixture?

Reply:
usage of retrieve_latest_ami other replace is createami test, in that test, we don't have cfn template, so I think we can't use this fixture